std/haxe/io/Bytes.hx

```diff
@@ -539,6 +539,22 @@ class Bytes {
 		return new Bytes(b.length,b);
 		#end
 	}
+	
+	public static function ofHex( hex:String ) : Bytes {
```

Comment:
Also, this method should be added to `js/_std/haxe/io/Bytes.hx`, `hl/_std_haxe/io/Bytes`, `php/_std_haxe/io/Bytes` and `eval/_std_haxe/io/Bytes` because these targets have their own implementations of `haxe.io.Bytes`

Reply:
I did all changes. All modes ( including CTR, CFB and OFB works) .  How exactlyshoukld  look tests for all public methods ? Also for Bytes.ofHex I'm not sure how to test different targets ( including hl with compilation from source)